src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -21,7 +21,7 @@ public class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the FART book. "
```

Comment:
If we changed this I think we need to change many other lines to be aligned - maybe we could leave it as address book, as it is still an address book after all?

Reply:
Are there many changes to this portion? Thought that if we were to be consistent with our naming, we could change everything to FART book but if not I can revert this change

Reply:
uhh i did have a few lines in the user guide and confirmation dialogs where it refers to it as fart book, but we can rectify that as an alpha bug

Reply:
Let's standardise naming